src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditorViewModel.java

```diff
@@ -0,0 +1,55 @@
+package org.jabref.gui.fieldeditors;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.util.StringConverter;
+
+import org.jabref.gui.util.BindingsHelper;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.EntryLinkList;
+import org.jabref.model.entry.ParsedEntryLink;
+
+public class LinkedEntriesEditorViewModel extends AbstractEditorViewModel {
+
+    private final BibDatabaseContext databaseContext;
+    private final ListProperty<ParsedEntryLink> linkedEntries;
+
+    public LinkedEntriesEditorViewModel(BibDatabaseContext databaseContext) {
+        this.databaseContext = databaseContext;
+        linkedEntries = new SimpleListProperty<>(FXCollections.observableArrayList());
+        BindingsHelper.bindContentBidirectional(
+                linkedEntries,
+                text,
+                EntryLinkList::serialize,
+                newText -> EntryLinkList.parse(newText, databaseContext.getDatabase()));
+    }
+
+    public ListProperty<ParsedEntryLink> linkedEntriesProperty() {
+        return linkedEntries;
+    }
+
+    public StringConverter<ParsedEntryLink> getStringConverter() {
+        return new StringConverter<ParsedEntryLink>() {
+            @Override
+            public String toString(ParsedEntryLink linkedEntry) {
+                if (linkedEntry == null) {
+                    return "";
+                }
+                return linkedEntry.getKey();
+            }
+
+            @Override
+            public ParsedEntryLink fromString(String key) {
+                return databaseContext.getDatabase().getEntryByKey(key).map(ParsedEntryLink::new).orElse(null);
+            }
+        };
+    }
+
+    public void jumpToEntry(ParsedEntryLink parsedEntryLink) {
+        // TODO: Implement jump to entry
```

Comment:
Ok that policy is very understandable. In the current case, I'm not sure if we really should remove it. The "jump to entry" feature was present in previous versions and this PR removes it (temporarily). I don't see a nice way to re-implement it right now as we have no good interface to control the focus of the main table (I really don't want to directly use the `JabRefFrame` class). But removing the comment here and above results also in a change of the language files and the "Jump to entry" string has to be readded by the translators at some point in the future. 

Should I still remove it? I don't have a strong opinion about it.